installation/GRCh38_related_files/calc_gc_content.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import argparse
+import re
+
+assert sys.version_info.major >= 3, "Python 3 is required for string operations" 
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-i", "--input", help="Input sequence - fasta file format", required=True)
+parser.add_argument("-o", "--output", help="Output file name", default='gc_content.txt')
+parser.add_argument("-t", "--threshold", help="Threshold", default=0.0)
+parser.add_argument("--stepsize", help="Threshold (regions with gc lower than threshold are discarded)", default=10000)
+parser.add_argument("-v", "--verbose", action = 'store_true', help="Add verbosity")
+args = parser.parse_args()
+
+threshold = float(args.threshold)
+stepsize = int(args.stepsize)
+
+output_data = ['\t'.join(['chromosome', 'start', 'end', 'gc_content'])]
+
+if args.verbose: print('Load data')
+with open(args.input, "r") as f:
+    data = f.read().splitlines()
+
+if args.verbose: print('Calculate chromosome stats')
+chr_names, chr_borders = [], []
+
+for i in range(len(data)):
+    if '>' in data[i]:
+        chr_names.append(data[i].split('>')[1].split(' ')[0])
+        chr_borders.append(i)
+chr_borders = chr_borders + [len(data)]
```

Comment:
Pretty unclear. chr_borders is the line-number (zero-based) in the input file, right? Maybe rename to `chr_last_line_idxs`?